assets/preload/weeks/week6.json

```diff
@@ -19,4 +19,4 @@
 
 	"hideStoryMode": false,
 	"hideFreeplay": false
-}
\ No newline at end of file
+}
```

Comment:
Unnecessary line ending change.

Reply:
these week files came by another PR i was doing, i've accidentally uploaded the files at the same branch as the pixel notesplashes. So, it will show these files with a line ending change